src/cargo/ops/cargo_generate_lockfile.rs

```diff
@@ -91,6 +91,35 @@ pub fn update_lockfile(manifest_path: &Path,
                                                   Method::Everything,
                                                   Some(&previous_resolve),
                                                   Some(&to_avoid)));
+
+    // Summarize what is changing for the user.
+    let print_change = |status: &str, msg: String| {
+        opts.config.shell().status(status, msg)
+    };
+    for (removed, added) in compare_dependency_graphs(&previous_resolve, &resolve) {
+        if removed.len() == 1 && added.len() == 1 {
+            if removed[0].source_id().is_git() {
+                try!(print_change("Updating", format!("{} #{} -> #{}",
+                    removed[0].name(),
```

Comment:
Doing that makes the update line look like this:
```
    Updating offscreen_gl_context v0.1.0 (https://github.com/ecoal95/rust-offscreen-rendering-context#9efc32eb) -> #9704865b
```
Which I don't think is better, personally.